src/layouts/MainLayout/components/AppLink.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <router-link :to="to" custom v-slot="{ isExactActive, navigate }" exact>
```

Comment:
`isExactActive` is not in prop or in any logic on the file, what it is for?

Reply:
v-slot is to get props that come from a component, to use on component children. In this case, the `<router-link>` component is returning some props to help me to get a custom router-link.

isExactActive is a boolean variable that is true when the route is exact that the route that I'm passing from `:to="to"`, if `:to="/home"` and the route where the user is, is exactly `localhost/home`, then the isExactActive variable will be true, if not, will be false

https://runebook.dev/es/docs/vue_router/api/index#router-link-s-v-slot